enforcer_cached.go

```diff
@@ -15,15 +15,19 @@
 package casbin
 
 import (
+	"errors"
 	"strings"
 	"sync"
 	"sync/atomic"
 )
 
+var ErrNoSuchKey = errors.New("there's no such key existing in cache")
```

Comment:
It would be even better if we can put all the exposed errors in casbin together in a package.